Add Bluetooth 2.0.0

Outpox opened the pull request:
# Bluetooth

Formerly known as SDH-Bluetooth, this plugin allows you to quickly connect and disconnect an already paired bluetooth devices.  

## Checklist:

- [x] I am the original author of this plugin.
- [x] I made my code efficient and readable.
- [x] I have verified that my plugin works properly on the latest versions of SteamOS and decky-loader.
- [x] I have verified that my plugin is unique and does not have the same functionality as another plugin on the store.

nabel0 reviewed (approved):
Paireda PlayStation4 Controller via bluetooth .
With this plugin is was able to disconnect and reconnect to controller while in game.
Everything is working as expected.

Titaniumtown reviewed (approved):
Would love to see this added!

Outpox commented:
As requested I added a license file (I chose the default one) and I also updated the version to 2.0.0 to remain compatible with my older legacy releases 1.0.0 and 1.1.0

TrainDoctor reviewed (approved):
👍🏻 
So long as the plugin build succeeds, I'll merge this shortly.

EMERALD0874 reviewed (approved):
Tested with a few devices, LGTM
